account_easy_reconcile/__openerp__.py

```diff
@@ -54,7 +55,6 @@
 """,
```

Comment:
As you moved description in `README.rst` you must remove the key and value in `__openerp__.py` otherwise this one will be displayed on apps.odoo.com and in Odoo. Plus we don't want the description being duplicated